source/grid/cell_id.cc

```diff
@@ -97,6 +97,14 @@ CellId::CellId(const CellId::binary_type &binary_representation)
 
 
 
+CellId::CellId(const std::string &string_representation)
+{
+  std::istringstream ss(string_representation);
+  ss >> *this;
```

Comment:
Stupid question. How should I read these two lines?

Reply:
102: We create an output stream from the buffer specified by the string representation.
103: We write the contents of the string to the cell id with the relevant overload of `>>` (i.e, operator>>(std::istream &, CellId &)`

Reply:
Thanks @drwells !